rudder-core/src/main/scala/com/normation/cfclerk/services/AsyncDeployAgent.scala

```diff
@@ -0,0 +1,7 @@
+package com.normation.cfclerk.services
+
+trait AsyncDeployAgent {
```

Comment:
Rename the trait AsyncDeploymentAgent, put in the correct file ( rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentAgent.scala )